.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,75 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.195.0/containers/rust
+{
+	"name": "thin-edge.io",
+	// Must be set to false, otherwise the cmd (or entrypoint) will not be PID 1 which is required by systemd
+	"overrideCommand": false,
+
+	// Docker Compose
+	// The 'service' property is the name of the service for the container that VS Code should
+	// use. Update this value and .devcontainer/docker-compose.yml to the real service name.
```

Comment:
Is it necessary to update this value? Else better to mention that it's optional.